theano/gpuarray/dnn.py

```diff
@@ -286,10 +286,13 @@ def c_lib_dirs(self):
     def do_constant_folding(self, node):
         return False
 
-    def __init__(self, border_mode, subsample=(1, 1), conv_mode='conv',
-                 precision="float32"):
+    def __init__(self):
         COp.__init__(self, ["conv_desc.c"], "APPLY_SPECIFIC(conv_desc)")
 
+    def make_node(self, kern_shape, border_mode, subsample=(1, 1), conv_mode='conv', precision="float32"):
+        if kern_shape.type.ndim != 1 or kern_shape.type.dtype != 'int64':
+            raise TypeError('kern must be 1D shape tensor')
```

Comment:
This message was present in the codebase even before I got to work. What do you suggest the message should be?